### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Education Data Migration, WEAMS Public

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#101873]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [x] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delayed feedback for this! I had some unexpected time out of office last week and I'm only now catching up. Please feel free to reach out via Slack if you have any questions about my feedback, or if my delay caused any issues for the team.
- I spotted several updates based on feedback I gave at design intent. Thank you for making those changes!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** When you use a table for any purpose, make sure the table has a title (the table-title prop in the va-table component). As a screen reader user, navigating a table can be a huge pain, and the title provides a summary of what's included in the table before you have to start exploring it so you can decide if you want to skip it or dig into it.
- [x] **Should:** Echoing a lot of the discussion from the design and IA folks, there are a lot of accessibility benefits to a user flow that only goes in one direction --- meaning, if a user has already moved on from a form input, we're not asking them to go back to a previous input (or pay attention to it for changes). Partly this is for cognitive load reasons, keeping the user focused on what they're doing now and not juggling multiple things. Partly this is for assistive technology reasons, as navigating back and/or notifying users of changes to something they've already passed can be challenging. 
   
   I think the search flow may have more success if you explore either:
   - A conditional reveal of the license/certification input, so that it only becomes visible after a state has been selected. (I don't normally love conditional reveals but there's a good use case for it here.)
   - Removing the state input from the search tool and instead allowing for filtering by state on the results page. (This may be less desirable for how you expect users to interact with the product.
- [x] **Must:** When [viewing search results](https://www.figma.com/design/xAydfsAlVVmhnFQA9MkeYw/Design-for-WEAMS-Public-in-CT-%26-SOB?node-id=3042-484160&t=8oDawOpNAR6NsBVI-4), the "Back to search" link currently floats off to the right-hand side. Screen magnification users will often start along the left-hand side of the main column of content and work their way down, only moving to the right-hand side of the screen if there's some kind of visual element indicating they should explore that region. If you don't know the "Back to search" link is over there, there's not really a reason to go looking in that area for it. I'd recommend moving the link to the left.
- [x] **Must:** When you have multiple headings at the same heading level and with the same direct parent, they should be unique relative to each other to make it easy for screen reader users to differentiate them from each other. I'm thinking of the [case where there are licenses with the same name in multiple states](https://www.figma.com/design/xAydfsAlVVmhnFQA9MkeYw/Design-for-WEAMS-Public-in-CT-%26-SOB?node-id=3510-94530&t=8oDawOpNAR6NsBVI-4). Visually, it's clear from the cards that each license goes with a different state. But if I'm a screen reader user navigating by heading, I hear "Barber" multiple times without necessarily realizing there's a state distinction. A few approaches for doing this:
   - Include the state in the heading text (eg. `Barber (Idaho)`)
   - Group the state as part of the heading text semantically, but visually style it as regular text in the card (`<h2><span class="looks-like-heading`>Barber</span><span class="looks-like-regular-text">Idaho</span></h2>`)
   - Associate the state with the heading using `aria-describedby` (`<h2 aria-describedby="some-id">Barber</h2><p id="some-id">Idaho</p>`).

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by cassidy-beach** [comment]
**Must: When [viewing search results](https://www.figma.com/design/xAydfsAlVVmhnFQA9MkeYw/Design-for-WEAMS-Public-in-CT-%26-SOB?node-id=3042-484160&t=8oDawOpNAR6NsBVI-4), the "Back to search" link currently floats off to the right-hand side.**
Deleted "Back to search" link on the right side of the page
![Image](https://github.com/user-attachments/assets/ce5f659a-0d6b-41a8-a48e-25194f39fffb)

**Comment by cassidy-beach** [comment]
**Should: Echoing a lot of the discussion from the design and IA folks, there are a lot of accessibility benefits to a user flow that only goes in one direction**
Simplified search and user flow
Removed the state input from the search tool and instead allowing for filtering by state on the results page\
![Image](https://github.com/user-attachments/assets/1b464fc7-62f1-4d25-9db7-50427668e569)

**Comment by cassidy-beach** [comment]
**Must: When you have multiple headings at the same heading level and with the same direct parent, they should be unique relative to each other to make it easy for screen reader users to differentiate them from each other.**
We have the state in the body
This is how the cards are formatted based on VADS
![Image](https://github.com/user-attachments/assets/29ec1c0b-7d7d-44d8-9018-72379a20dfbd)

**Comment by cassidy-beach** [comment]
**Must: When you use a table for any purpose, make sure the table has a title (the table-title prop in the va-table component).** 
"Test info" is the title the business wants 
![Image](https://github.com/user-attachments/assets/72cd06b7-59d5-4d14-b6d1-65ae3c34b409)
